mne_bids/write.py

```diff
@@ -1053,7 +1055,7 @@ def write_raw_bids(raw, bids_basename, bids_root, events_data=None,
 
         Note that the modality 'meg' is automatically inferred from the raw
         object and extension '.fif' is copied from raw.filenames.
-    bids_root : str
+    bids_root : str | Path
```

Comment:
also update the docstring of `read_raw_bids` ?

Reply:
Will do, see https://github.com/mne-tools/mne-bids/pull/362#issuecomment-600842349